### PR TITLE
lakectl: deprecate dbt and metastore commands

### PR DESCRIPTION
**Pull request by nopcoder**
All lakectl `dbt` and `metastore` command will print deprecation warnings.

Close https://github.com/treeverse/lakeFS/issues/6564

### CODE REVIEW

**Comment by itaiad200** [comment]
I think that the integration pages should help clarify the current situation: These command are deprecated and would be removed soon. A better, automated integration will replace them.

**Review by itaiad200 — APPROVED** [review]
LGTM

**Comment by nopcoder** [comment]
@itaiad200 please review the documentation changes

File: docs/integrations/glue_hive_metastore.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ redirect_from: /using/glue_hive_metastore.html
 
 # Using lakeFS with the Glue/Hive Metastore
 
+{: .warning }
+**Deprecated Feature:** This feature is being phased out. A superior replacement will be introduced soon.
```
Comment (itaiad200) [inline docs/integrations/glue_hive_metastore.md:11]: These warnings are vague and don't give confidence that we'll continue to support these integrations. 
```suggestion
**Deprecated Feature:** Having heard the feedback from the community, we are planning to replace the below manual steps with an automated process. You can read more about it [here](https://github.com/treeverse/lakeFS/issues/6461).
```

Comment (itaiad200) [reply docs/integrations/glue_hive_metastore.md:11]: Here and elsewhere

**Review by itaiad200 — CHANGES_REQUESTED** [review]

**Review by itaiad200 — APPROVED** [review]
LGTM

**Pull request merged**